packages/rocketchat-importer-csv/server.js

```diff
@@ -254,7 +298,7 @@ Importer.CSV = class ImporterCSV extends Importer.Base {
 							const creator = this.getUserFromUsername(msg.username);
 							if (creator) {
 								const msgObj = {
-									_id: `csv-${ csvChannel.id }-${ msg.ts }`,
+									_id: `csv-${ csvChannel.id }-${ roomCounter }-${ msg.ts }`,
```

Comment:
@graywolf336 Is my explanation satisfactory, or should I rewrite the code to store the last timestamp and add a suffix on dup timestamps, so old imports won't ever have the chance to be duplicated? Or even store an array of seen timestamps...otherwise the code is dependent on the csv line order to be time-based.